src/app/public/modules/flyout/flyout.component.html

```diff
@@ -22,6 +25,10 @@
     [attr.aria-valuenow]="flyoutWidth"
     [max]="config.maxWidth"
     [min]="config.minWidth"
+    [ngClass]="{
+      'sky-flyout-hidden': !isOpen && !isOpening,
+      'sky-flyout-fullscreen': isFullscreen
```

Comment:
`.sky-flyout-hidden` doesn't actually have any style declarations for this `.sky-flyout-resize-handle` element. Did you mean to put it here?

On second thought, do we really need these two lines at all? We could infer the input should take different styles in the parent `div` `ngClass` triggers. For example, you could rewrite your scss to look like this:
```
.sky-flyout {
 ...
  &.sky-flyout-fullscreen {
    min-width: 100%;
    max-width: 100%;

      .sky-flyout-resize-handle {
        // CSS for handle when fullscreen
      }
  }
  &.sky-flyout-hidden {
    visibility: hidden;

      .sky-flyout-resize-handle {
        // CSS for handle when hidden
      }
  }
...
```

Reply:
Good call. Done